website/docs/metadata_indexing.md

```diff
@@ -10,18 +10,65 @@ The [pluggable indexing subsystem](https://www.onehouse.ai/blog/introducing-mult
 of Hudi depends on the metadata table. Different types of index, from `files` index for locating records efficiently
 to `column_stats` index for data skipping, are part of the metadata table. A fundamental tradeoff in any data system
 that supports indices is to balance the write throughput with index updates. A brute-force way is to lock out the writes
-while indexing. However, very large tables can take hours to index. This is where Hudi's novel asynchronous metadata
-indexing comes into play.
+while indexing. Hudi supports index creation using SQL, Datasource as well as async indexing. However, very large tables 
```

Comment:
please cover how concurrency control is done.. Just write out index building protocols as simple bullets.

Reply:
Addressed